poi/src/test/java/org/apache/poi/ss/formula/functions/TestAverageIf.java

```diff
@@ -107,16 +108,32 @@ void testExample1() {
         };
 
         confirmError(ErrorEval.DIV_ZERO, args);
-        
+
         // "=AVERAGEIF(A2:A5, "<95000", B2:B5 )"
         args = new ValueEval[]{
                 EvalFactory.createAreaEval("A2:A5", a2a5),
                 new StringEval(">250000"),
                 EvalFactory.createAreaEval("B2:B5", b2b5)
         };
-        confirm( 24500, args);
-/*
-*/        
+        confirm(24500, args);
+    }
+
+    @Disabled("broken test")
```

Comment:
I added this test and it shows that the new code cannot handle M$ example 2 - I may look at the fixing this myself but more likely I will delete all the new code because it is incomplete